### PR TITLE
traducao: security-basic-authentication-howto.adoc.po

### LINKED ISSUE #16: Translation: security-basic-authentication-howto
Hello everybody
I am working on translating this guide: security-basic-authentication-howto

Note: is this my first contribution to an open source project!

### PR DESCRIPTION
**Pull request by Renata00000**
Hello everyone, I kindly request a review.
fixed #16

### CODE REVIEW

File: l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po [hunk 1]
```diff
@@ -24,9 +24,8 @@ msgstr "Ativar a autenticação básica"
 
 #. type: Plain text
 #: upstream/_guides/security-basic-authentication-howto.adoc:8
-#, fuzzy
 msgid "Enable xref:security-basic-authentication.adoc[Basic authentication] for your Quarkus project and allow users to authenticate with a username and password."
-msgstr "Active  link:security-basic-authentication.html[a autenticação básica] para o seu projeto Quarkus e permita que os utilizadores se autentiquem com um nome de utilizador e uma palavra-passe."
+msgstr "Active  link:security-basic-authentication.html[a autenticação básica] para o seu projeto Quarkus e permita que os usuários se autentiquem com um use rname e um password."
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po:28]:
```suggestion
msgstr "Ative link:security-basic-authentication.html[a autenticação básica] para o seu projeto Quarkus e permita que os usuários se autentiquem com um username e um password."
```

File: l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po [hunk 2]
```diff
@@ -36,9 +35,8 @@ msgstr "Pré-requisitos"
 
 #. type: Plain text
 #: upstream/_guides/security-basic-authentication-howto.adoc:12
-#, fuzzy
 msgid "You have installed at least one extension that provides an `IdentityProvider` based on username and password, such as xref:security-jdbc.adoc[Elytron JDBC]."
-msgstr "Instalou pelo menos uma extensão que fornece um `IdentityProvider` baseado no nome de utilizador e na palavra-passe, como o  link:security-jdbc.html[Elytron JDBC]."
+msgstr "Você Instalou pelo menos uma extensão que fornece um `IdentityProvider` baseado no usename e no password, como o  link:security-jdbc.html[Elytron JDBC]."
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po:39]:
```suggestion
msgstr "Você instalou pelo menos uma extensão que fornece um `IdentityProvider` baseado no usename e no password, como o  link:security-jdbc.html[Elytron JDBC]."
```

File: l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po [hunk 3]
```diff
@@ -78,27 +75,23 @@ msgstr ""
...
+msgstr "Para obter informações sobre outros métodos que você pode utilizar para configurar as credenciais de usuário necessárias, consulte a seção  link:security-testing.html#configuring-user-information[Configurar as informações do usuario] do guia \"Teste de segurança\" do Quarkus."
 
 #. type: delimited block =
 #: upstream/_guides/security-basic-authentication-howto.adoc:43
-#, fuzzy
 msgid "Configuring user names, secrets, and roles in the `application.properties` file is only suitable for testing scenarios.  If you are securing a production application, always use a database to store this information."
-msgstr "A configuração de nomes de utilizador, segredos e funções no ficheiro `application.properties` só é adequada para cenários de teste. Se estiver a proteger uma aplicação de produção, utilize sempre uma base de dados para armazenar estas informações."
+msgstr "A configuração de nomes de username, secrets e roles no arquivo `application.properties` só é adequada para cenários de teste. Se estiver protegerndo uma aplicação em produção, utilize sempre uma base de dados para armazenar estas informações."
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po:94]:
```suggestion
msgstr "A configuração de nomes de username, secrets e roles no arquivo `application.properties` só é adequada para cenários de teste. Se estiver protegendo uma aplicação em produção, utilize sempre uma base de dados para armazenar estas informações."
```

File: l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po [hunk 4]
```diff
@@ -108,9 +101,8 @@ msgstr "Próximas etapas"
 
 #. type: Plain text
 #: upstream/_guides/security-basic-authentication-howto.adoc:48
-#, fuzzy
 msgid "For a more detailed walk-through that shows you how to configure Basic authentication together with Jakarta Persistence for storing user credentials in a database, see the xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication and Jakarta Persistence] guide."
-msgstr "Para obter um guia mais detalhado que mostra como configurar a autenticação básica juntamente com a persistência Jakarta para armazenar credenciais de utilizador numa base de dados, consulte o guia  link:security-basic-authentication-tutorial.html[Proteger uma aplicação Quarkus com autenticação básica e persistência Jakarta]."
+msgstr "Para obter um guia mais detalhado que mostra como configurar a autenticação básica juntamente com o Jakarta Persistence para armazenar credenciais de utilizador numa base de dados, consulte o guia  link:security-basic-authentication-tutorial.html[Proteger uma aplicação Quarkus com autenticação básica e o Jakarta Persistence]."
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po:105]:
```suggestion
msgstr "Para obter um guia mais detalhado que mostra como configurar a autenticação básica juntamente com o Jakarta Persistence para armazenar credenciais de usuário numa base de dados, consulte o guia  link:security-basic-authentication-tutorial.html[Proteger uma aplicação Quarkus com autenticação básica e o Jakarta Persistence]."
```

File: l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po [hunk 5]
```diff
@@ -132,9 +124,8 @@ msgstr "link:security-identity-providers.html[Fornecedores de identidade]"
 
 #. type: Plain text
 #: upstream/_guides/security-basic-authentication-howto.adoc:54
-#, fuzzy
 msgid "xref:security-testing.adoc#configuring-user-information[Configuring User Information in application.properties]"
-msgstr "link:security-testing.html#configuring-user-information[Configurar as informações do utilizador em application.properties]"
+msgstr "link:security-testing.html#configuring-user-information[Configurar as informações do usuario no application.properties]"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-basic-authentication-howto.adoc.po:128]:
```suggestion
msgstr "link:security-testing.html#configuring-user-information[Configurar as informações do usuário no application.properties]"
```

**Review by gastaldi — CHANGES_REQUESTED** [review]
Found some minor typos, but looks good overall, good job!

**Comment by gastaldi** [comment]
Don't forget to review *all* entries and remove the `fuzzy` keyword above the msgid (disable the `Needs Some Work` toggle in Poedit) when it's ready

**Comment by Renata00000** [comment]
TKS George!

**Review by gastaldi — APPROVED** [review]

**Pull request merged**